docs/UserGuide.md

```diff
@@ -422,13 +422,17 @@ Format: `find -e KEYWORD`
 Examples:
 * `find -e Meeting` returns `CS2103 Meeting` and `CS2101 meeting`.
 * `find -e seminar` returns `CS Seminar` and `seminar 1`.
+* `find -e dinner` returns `Family Dinner`.
+![Find Event Behaviour](images/ug-images/eventBehaviourImages/find-event-behaviour.png)
```

Comment:
Think there should be a new line between these 2 lines as per the other pictures.

Reply:
Have made the edit.